ebu_tt_live/documents/ebuttd.py

```diff
@@ -46,8 +52,23 @@ def create_from_raw_binding(cls, binding):
         instance._ebuttd_content = binding
         return instance
 
+    def _get_bds(self):
```

Comment:
Er right, okay, and this is an internal facing method, so not a big deal? I'd just never heard of it, but I have heard of XML and DOM, which are in the other two similar getters.